index.js

```diff
@@ -0,0 +1,39 @@
+
+const checkIfNum = (value) => {
+  return !isNaN(value) && value !== ''
+}
+
+const checkForOperand = (value) => {
+  const operands = ['+', '-', '*', '/']
+
+  return checkIfNum(value) || operands.includes(value)
+}
+
+const doOperation = (numbers) => {
+  switch (numbers[1]) {
+    case '+':
+      return numbers[0] + numbers[2]
```

Comment:
You should be typecasting here. It's possible your array coming into this function is `[2 + '2']` which would result in `'22'` not `4`.